content/en/blog/2022/steering-election-results/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "2022 Istio Steering Committee Election Results"
+subtitle: Announcing the newest Istio Steering Committee members.
```

Comment:
Do we need both description and subtitle?

(I assume the blog from 2020 had both)